js/toast.js

```diff
@@ -0,0 +1,14 @@
+let toastO=document.getElementById('toastO');
+let toastI=document.getElementById('toastI');
+
+let toastMessage=(message,backgroundColor)=>{
+    if(backgroundColor){
+        toastI.style.background=backgroundColor;
+    }
+    toastI.innerHTML=message;
+    toastO.style.display='flex';
```

Comment:
no inline js styles unless required()not possible through css

and if it is really not possible through css then something is wrong, change the structure